blog/docs/releases/announcing-knative-v1-3-release.md

```diff
@@ -0,0 +1,207 @@
+
+#### 🐞 Bug Fixes
+
+- Added support for Brokers with long namespace and name values. ([#1971](https://github.com/knative-sandbox/eventing-kafka-broker/pull/1971))
+- KafkaChannel reconciler checks for empty subscriber URI. ([#1905](https://github.com/knative-sandbox/eventing-kafka-broker/pull/1905))
+
+#### Known issues
+
+- The reference information for metrics is not built successfully for Kafka channels. ([#1824](https://github.com/knative-sandbox/eventing-kafka-broker/pull/1824))
+
+### RabbitMQ Broker and Source v1.3
+
+<!-- Original notes are here: https://github.com/knative-sandbox/eventing-rabbitmq/releases/tag/knative-v1.3.0 -->
+
+#### 💫 New Features & Changes
+
+- Added publisher confirms to ingress. Return HTTP Status 200 only when RabbitMQ confirms receiving and storing the message. See issue [#334](https://github.com/knative-sandbox/eventing-rabbitmq/issues/334). ([#568](https://github.com/knative-sandbox/eventing-rabbitmq/pull/568))
+- The Source Adapter and Broker Dispatcher's Prefetch Count behavior is now the same. Updated the Trigger's webhook to validate:
+    - Has a default value of 1. FIFO behavior
+    - Have limits: 1 ≤ prefetchCount ≤ 1000 ([#536](https://github.com/knative-sandbox/eventing-rabbitmq/pull/536))
+- All core Knative Eventing RabbitMQ pods should now be able to run in the restricted pod security standard profile. ([#541](https://github.com/knative-sandbox/eventing-rabbitmq/pull/541))
+- Various refactorings and code health improvements. ([#552](https://github.com/knative-sandbox/eventing-rabbitmq/pull/552), [#572](https://github.com/knative-sandbox/eventing-rabbitmq/pull/572))
+- Makefile-based worklow, includes migrating GitHub Actions. ([#525](https://github.com/knative-sandbox/eventing-rabbitmq/pull/525), [#569](https://github.com/knative-sandbox/eventing-rabbitmq/pull/569), [#579](https://github.com/knative-sandbox/eventing-rabbitmq/pull/579))
+- Improved Broker and Source README docs and Samples description and files. ([#555](https://github.com/knative-sandbox/eventing-rabbitmq/pull/555))
+
+#### 🐞 Bug Fixes
+
+- Removing the dead letter sink on a trigger will now properly fall back to the Broker's dead letter sink, if one is defined. ([#533](https://github.com/knative-sandbox/eventing-rabbitmq/pull/533))
+- Configuring messages sent into the RabbitMQ Broker to be persistent as the Queues used by the Broker are always durable.
+Now if the user set the configuration of the RabbitMQ Source Exchange and Queue to be durable, the messages are also durable. ([#560](https://github.com/knative-sandbox/eventing-rabbitmq/pull/560))
+
+
+## Client v1.3
+
+<!-- Original notes are here: https://github.com/knative/client/blob/main/CHANGELOG.adoc#v130-2022-03-08 -->
+
+### 💫 New Features & Changes
+
+- Added Knative Eventtype support. ([#1598](https://github.com/knative/client/pull/1598))
```

Comment:
```suggestion
- Added Knative eventtype support. ([#1598](https://github.com/knative/client/pull/1598))
```

(also not 100% on the proper convention here, but looking at the PR docs/comments, seems to be lowercase most of the time)